python/pyarrow/types.pxi

```diff
@@ -847,6 +847,25 @@ cdef timeunit_to_string(TimeUnit unit):
         return 'ns'
 
 
+FIXED_OFFSET_PREFIX = '+'
+
+
+def tzinfo_to_string(tz):
```

Comment:
These two functions would deserve a docstring, and appropriate unit tests.

Reply:
Done!